lib/icinga/clusterevents-check.cpp

```diff
@@ -88,9 +88,9 @@ static void SendEventExecuteCommand(const Dictionary::Ptr& params, long exitStat
 	executedParams->Set("start", start);
 	executedParams->Set("end", end);
 
-	if (origin->IsLocal()) {
-		ClusterEvents::ExecutedCommandAPIHandler(origin, executedParams);
-	} else {
+	ClusterEvents::ExecutedCommandAPIHandler(origin, executedParams);
+
+	if (!origin->IsLocal()) {
```

Comment:
Please revert this (just locally for now) and try to solve this problem by passing origin=nullptr to `listener->RelayMessage()` in `ClusterEvents::ExecutedCommandAPIHandler()`.